Source/Fuse.Launcher.InAppBrowser/InAppBrowserLauncher.uno

```diff
@@ -0,0 +1,71 @@
+using Uno;
+using Uno.UX;
+using Uno.Threading;
+using Uno.Compiler.ExportTargetInterop;
+
+namespace Fuse.LauncherImpl
+{
+	public static class InAppBrowserLauncher
+	{
+		public static void LaunchInAppBrowser(string url)
+		{
+			if defined(iOS)
+				InAppBrowseriOSImpl.OpenUrl(url);
+			if defined(Android)
+				InAppBrowserAndroidImpl.OpenUrl(url);
+		}
+
+	}
+
+	extern(iOS) class InAppBrowseriOSImpl
+	{
+
+		public static void OpenUrl(string url)
+		{
+			Safari.PresentSafari(url);
+		}
+
+	}
+
+	extern(Android) class InAppBrowserAndroidImpl
+	{
+
+		public static void OpenUrl(string url)
+		{
+			Chrome.PresentChrome(url);
+		}
+
+	}
+
+	[Require("Xcode.Framework", "SafariServices")]
+	[ForeignInclude(Language.ObjC, "AVFoundation/AVFoundation.h")]
+	[Require("Source.Include", "UIKit/UIKit.h")]
+	[Require("Source.Include", "SafariServices/SafariServices.h")]
+	public extern(iOS) class Safari
+	{
+		[Foreign(Language.ObjC)]
+		public static void PresentSafari(string url)
+		@{
+			NSURL* u = [[NSURL alloc] initWithString: url];
+			SFSafariViewController* vc = [[SFSafariViewController alloc] initWithURL: u entersReaderIfAvailable:NO];
+			dispatch_async(dispatch_get_main_queue(), ^{
+				[[UIApplication sharedApplication].keyWindow.rootViewController presentViewController:vc animated:YES completion:nil];
+			});
+		@}
+
+	}
+
+	[Require("Gradle.Dependency.Implementation", "androidx.browser:browser:1.3.0")]
```

Comment:
Change to `androidx.browser:browser:1.0.0` to make AppVeyor build pass. See https://github.com/fuse-open/fuselibs/pull/1412#issuecomment-967233892.